__tests__/app.test.js

```diff
@@ -0,0 +1,42 @@
+const request = require("supertest");
+const testData = require("../db/data/test-data");
+const seed = require("../db/seeds/seed");
+const { app } = require("../app");
+const db = require("../db/connection")
+
+afterAll(() => {
+  if (db.end) {
+    return db.end()}
+});
+beforeEach(() => seed(testData));
+
+describe("Standard error messages", () => {
+  it("Returns a 404 error when given a bad request", () => {
+    return request(app)
+      .get("/api/banana")
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("No such path found. Try again...");
+      });
+  });
+});
+
+describe("1. GET request.", () => {
+  test("sends back an array of games categories", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.category).toBeInstanceOf(Array);
```

Comment:
toBeInstanceOf is useful but obsolete in this case if you're using a forEach (which only works on Arrays) later on. Not a major issue though. 

What is more significant though is checking your array has length. Your expect statement is contained within a forEach, this wouldn't ever be reached by the empty array, and describe blocks with no expect statements pass as default, so it could cause false positives!